napari/layers/labels/labels.py

```diff
@@ -1253,7 +1264,7 @@ def _draw(self, new_label, last_cursor_coord, coordinates):
                 and self.data[tuple(np.round(c).astype(int))] == 0
             ):
                 continue
-            if self._mode in [Mode.PAINT, Mode.ERASE]:
+            if self._mode in [Mode.PAINT, Mode.ERASE, Mode.FILL_CONTOUR]:
```

Comment:
```suggestion
            if self._mode in {Mode.PAINT, Mode.ERASE, Mode.FILL_CONTOUR}:
```
small performance update